### PR TITLE
Laid out directory and files

### PR DESCRIPTION
**Pull request by FrankreedX**
After this is merged, rebase all ur branches

### CODE REVIEW

**Comment by solderq35** [comment]
Will close https://github.com/Golf-Drill-Challenge-App/Golf-App/pull/47 as it's now mostly redundant, but can refer to it for future reference. 

Login PR will be kept open as a draft, as we will look into integrating login some time in the future.

**Review by robwhess — COMMENTED** [review]
This looks fine to me.  I'll let it up to you all to approve/merge changes.

File: app/index.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+import {Link} from "expo-router";
 
+//This is the root file
 export default function Index() {
     return (
         <PaperProvider>
             <SafeAreaView style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
                     <Text>Open up App.js to start working on your app!</Text>
+                    <Link href={"/drill/"}>/drill/</Link>
+                    <Link href={"/drill/1234"}>/drill/1234 slug</Link>
```
Comment (solderq35) [inline app/index.js:13]: If we're certain about using bottom tabs navigation, we can consider doing that here: https://docs.expo.dev/router/advanced/tabs/#dynamic-routes

Similar to what Andy's navigation PR was doing.

**Comment by solderq35** [comment]
It could also be a good idea to agree on where any potential component files would go (maybe a `components` folder at the top level, as was done in last year's version: https://github.com/Golf-Drill-Challenge-App/Golf-App/tree/archived_app).

**Comment by solderq35** [comment]
I think I prefer git merge to git rebase, since with rebase you need to fix merge errors on multiple commits. If we squash merge all the PR's the main branch's commit history will look ok anyways.

File: app/index.js (hunk 1 above)
Comment (FrankreedX) [reply app/index.js:13]: Ye I agree

Comment (FrankreedX) [reply app/index.js:13]: Although I also want the navbar to appear on some of the tabs when they're active. Maybe we need to extract some index.js files out of the folder for tabs?

**Comment by FrankreedX** [comment]
> I think I prefer git merge to git rebase, since with rebase you need to fix merge errors on multiple commits. If we squash merge all the PR's the main branch's commit history will look ok anyways.
> 
> Otherwise, I think it looks good enough to merge into layout branch. Further work can be done on separate branches.

iirc in my old workplace, git rebase is used to add changes from upstream to your branch, to keep your history clean and make it easier to review. Then git merge is used when it is merged into main

**Pull request merged**